core/state/contracts/poly.sol

```diff
@@ -1,3 +1,4 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Why the MIT licence? What this code taken from somewhere else? I see that it was introduced by Alexey in PR #665, so it should have the same licence as the rest of Erigon, namely LGPL.